lisa/tools/fio.py

```diff
@@ -0,0 +1,142 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import re
+from decimal import Decimal
+from typing import List, Type, cast
+
+from lisa.executable import Tool
+from lisa.notifier import FIOResult
+from lisa.operating_system import Debian, Posix, Redhat, Suse
+from lisa.util import LisaException
+
+from .git import Git
+
+
+class Fio(Tool):
+    fio_repo = "https://github.com/axboe/fio/"
+    branch = "fio-3.29"
+    _result_pattern = re.compile(
+        r"([\w\W]*?)iops.*: min=(?P<min_iops>.+?), max=(?P<max_iops>.+?), "
+        r"avg=(?P<avg_iops>.+?), stdev=(?P<stdev_iops>.+?), "
```

Comment:
How about values like stdev_iops, it looks they are not sent in the message.

Reply:
db table doesn't have this column.